### ISSUE TITLE
Error with rust-nightly: type annotations needed for Box<_>

### ISSUE THREAD

**Issue by philipp8** [Drakulix/simplelog.rs#153]
Hey, I've been running into an issue with nightly build:
```
error[E0282]: type annotations needed for `Box<_>`
--> /usr/local/cargo/registry/src/index.crates.io-6f17d22bba15001f/time-0.3.25/src/format_description/parse/mod.rs:83:9
 |
 |     let items = format_items
 |         ^^^^^
 ...
```
found the issue here: https://github.com/time-rs/time/issues/681

Can we try bumping the version for time-rs?

**Comment by Drakulix** [comment]
> Can we try bumping the version for time-rs?

Sure, PRs welcome.

**Comment by philipp8** [comment]
Seems like `cargo update` fixed it now using `Updating time v0.3.25 -> v0.3.36`
